cmd/up.go

```diff
@@ -29,9 +29,21 @@ func UpCommand() cli.Command {
 			Usage: "Specify an alternate project name (default: directory name)",
 		},
 	}...)
+
+	cmd.Action = app.WithProject(factory, ProjectUp)
```

Comment:
Darren suggested this. So the idea is to override the action in order to inject the waiter in the cli part of the code (vs. in rancher-compose-executor code). So ultimately, we need to use the new `ProjectUp` function to inject the waiter.